packages/proxy/package.json

```diff
@@ -82,7 +82,7 @@
     "@anthropic-ai/sdk": "^0.39.0",
     "@apidevtools/json-schema-ref-parser": "^11.9.1",
     "@aws-sdk/client-bedrock-runtime": "^3.738.0",
-    "@braintrust/core": "^0.0.85",
+    "@braintrust/core": "workspace:*",
```

Comment:
this won't work for the proxy repo on its own, unfortunately. if you made `core` changes, then we'll need to land those, publish them, and then increment this independently.

Reply:
thanks! will make sure we do